worlds/noita/Options.py

```diff
@@ -7,11 +7,13 @@ class PathOption(Choice):
     Main Path includes the main 7 biomes you typically go through to get to the final boss.
     Side Path includes the Lukki Lair and Fungal Caverns. 9 biomes total.
     Main World includes the full world (excluding parallel worlds). 14 biomes total.
+    Beta Biome includes the Main World as well as the new beta biome. Note that it is fairly small.
     Note: The Collapsed Mines have been combined into the Mines as the biome is tiny."""
     display_name = "Path Option"
     option_main_path = 1
     option_side_path = 2
     option_main_world = 3
+    option_beta_biome = 5
```

Comment:
what's wrong with 4? :think:

Reply:
I'm slowly working on another update that's using the 4 slot, just don't want to steal the number at the moment.